changelog.md

```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## NEXT
+
+- Added the method `from_dict` for all node and edges.
+- Changed behaviour of `cai_causal_graph.causal_graph.CausalGraph.add_node` when both `identifier` and (`time_lag`, `variable_name`) are provided. Now, if all are provided, the method will raise an error only if `identifier` is not
+  equal to `get_name_with_lag(time_lag, variable_name)`, that is, the correct name.
```

Comment:
this looks wrong to me - you are changing `CausalGraph.add_node` method to account for `TimeSeriesCausalGraph` behaviour.

Reply:
I would actually agree that this is slightly worrying. Maybe it's just the messaging in this changelog that needs to be changed?